config/default.yml

```diff
@@ -1,3 +1,8 @@
+Minitest/DuplicatedTestRun:
+  Description: 'This cop detects when a test class has test methods, and its parent also has tests methods causing the parents tests to run them twice.'
```

Comment:
This reads a little awkwardly, how about:

```suggestion
  Description: 'This cop detects duplicate test runs caused by one test class inheriting from another.'
```

Reply:
Thanks! I agreed. I updated the description after moving the rule so that it's sorted alphabetically.

Reply:
It may be better to update the cop name to `Minitest/DuplicateTestRuns`. How about it?

Reply:
@andyw8 Do you have an opinion on this cop name?

Reply:
I'm leaning towards `DuplicateTestRun`. We're looking for duplicate _runs_ across the whole project, but in general we're detecting each duplicate _run_ individually.

Reply:
ok, I updated it to `DuplicateTestRun` and rebased it so that it now checks for sorted cop names in the `default` file. https://github.com/rubocop/rubocop-minitest/pull/165

Reply:
The name looks good. Thank you!